server/lib/scene/index.js

```diff
@@ -30,6 +33,7 @@ const SceneManager = function SceneManager(stateManager, event, device, message,
   this.queue = queue({
     autostart: true,
   });
+  this.sunCalc = sunCalc;
```

Comment:
Why do you put sunCalc here? you can require sunCalc in the file where it's needed :)

Reply:
This is what I had done before. But I couldn't mock the library. It is only by doing this that the mock of 'proxyquire' works

Reply:
oh right, true! Good for me :)